src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -242,19 +261,29 @@ public void apply(final VariantContext vc, final ReadsContext readsContext, fina
 
             validateVariantsOrder(vc);
 
+            final boolean thisIntervalIsReference =  vc.getAlternateAlleles().size() == 1 && vc.getAlternateAllele(0).equals(Allele.NON_REF_ALLELE);
+
             // GenomeLocSortedSet will automatically merge intervals that are overlapping when setting `mergeIfIntervalOverlaps`
             // to true.  In a GVCF most blocks are adjacent to each other so they wouldn't normally get merged.  We check
             // if the current record is adjacent to the previous record and "overlap" them if they are so our set is as
             // small as possible while still containing the same bases.
+            if (previousInterval != null && previousInterval.overlapsWithMargin(refInterval, 0) &&
+                    (previousIntervalIsReference || thisIntervalIsReference)) {
+                logger.warn("Current interval " + refInterval.toString() + " overlaps previous interval " + previousInterval.toString());
```

Comment:
I don't know what you think about this particular case... is it common that if some ref blocks overlap then all the ref blocks overlap and this will output thousands of messages to the command line? Perhaps this should be a captured OneShotLogger? Unless you think its helpful to get all of them...